internal/cmd/base/servers.go

```diff
@@ -560,7 +560,7 @@ func (b *Server) ConnectToDatabase(dialect string) error {
 	if b.DatabaseMaxOpenConnections == 1 {
 		return fmt.Errorf("unable to create db object with dialect %s: %s", dialect, "max_open_connections must be unlimited by setting 0 or at least 2")
 	} else {
-		underlyingDB, err := dbase.DB()
+		underlyingDB, err := dbase.SqlDB()
```

Comment:
likewise, the setting of MaxOpenConnections could be moved into an Option for the call to `db.Open`